app/helpers/helpers.js

```diff
@@ -0,0 +1,7 @@
+const bcryptjs = require('bcryptjs');
+const config = require('../../config');
+
+exports.encryptPassword = password => {
+  const encryptedPassword = bcryptjs.hashSync(password, Number(config.auth.rounds));
+  return encryptedPassword;
+};
```

Comment:
In this function is necessary to catch a possible error that could be from bcryptjs library so you need to use `try catch` block.